src/test/regress/expected/failure_drop_table.out

```diff
@@ -0,0 +1,234 @@
+SET citus.next_shard_id TO 100300;
+ALTER SEQUENCE pg_catalog.pg_dist_placement_placementid_seq RESTART 40;
+SELECT citus.mitmproxy('flow.kill()');
+ mitmproxy 
+-----------
+ 
+(1 row)
+
+SELECT citus.mitmproxy('recorder.reset()');
+ mitmproxy 
+-----------
+ 
+(1 row)
+
+-- none of these commands generate network traffic
+SET citus.shard_replication_factor TO 2;
+CREATE TABLE append (id int);
+SELECT create_distributed_table('append', 'id', 'append');
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+DROP TABLE append;
+-- prove no network traffic happend
+SELECT count(1) FROM citus.mitmproxy('recorder.dump()');
+ count 
+-------
```

Comment:
why do we see `1` if we're to prove no network traffic happened?

Reply:
This is a bug I haven't gotten around to fixing yet, it always returns an empty line at the end.

Reply:
Fixed: https://github.com/citusdata/citus/pull/2119/commits/a7e26cf44001ef1991dc0b59a9beccd611d5361f